src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -134,12 +137,27 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
       'dist',
       'codicon.css',
     ]);
-    const jsUri =
-        getUri(webview, this.context.extensionUri, ['media', 'CfgEditor', 'cfgeditor.js']);
+    const jsUri = getUri(webview, this.context.extensionUri, ['media', 'CfgEditor', 'index.js']);
 
     const cssUri =
         getUri(webview, this.context.extensionUri, ['media', 'CfgEditor', 'cfgeditor.css']);
 
+    const htmlPath =
+        vscode.Uri.joinPath(this.context.extensionUri, 'media/CfgEditor/cfgeditor.html');
```

Comment:
oh, we use also uri in vscode?

Reply:
Can I ask you about the meaning? :sweat_smile: 

- "Oh, we also use `vscode.Uri` directly in vscode?"
- "Oh, we also use URI class in vscode?"

Reply:
sorry for ambiguous expression.. 😞 

I'm asking that
- oh, we have to use also vscode.uri class for vscode.workspace.fs?
- if so, all of `nodejs.uri` vars for `nodejs.fs` should be replaced by `vscode.uri` for `vscode.workspace.fs`?

The reason why I'm asking it is...
- if we have to change from `nodejs.fs` to `vscode.workspace.fs`,
- we also have to change components that `vscode.workspace.fs` needs

Reply:
> we have to use also vscode.uri class for vscode.workspace.fs?

Yes, parameter of `vscode.workspace.fs` is `vscode.Uri`

> all of nodejs.uri vars for nodejs.fs should be replaced by vscode.uri for vscode.workspace.fs?

AFAIK, yes...!

> we also have to change components that vscode.workspace.fs needs

IIUC, this may be handled in #746